app/workers/users/record_field_test_event_worker.rb

```diff
@@ -0,0 +1,29 @@
+module Users
+  class RecordFieldTestEventWorker
+    include Sidekiq::Worker
+    include FieldTest::Helpers
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform(user_id, experiment, goal)
+      user = User.find(user_id)
+
+      if goal == "user_views_article_four_days_in_week"
+        determine_pageview_goal(user, experiment)
+      else
+        field_test_converted(experiment, participant: user, goal: goal)
```

Comment:
This class runs the risk of becoming a bottleneck that all experiments conversions run. If a second (unrelated) experiments is started, do you envision conversions running through this class, or another worker?

Reply:
I thought there's some positives to putting all the possible cases in one place to share logic where it makes sense, but I'm open to re-organizing this in any way that makes sense now or down the line.